Bottom: ec2abae37f0c911e055070a16ee91e1236725cb1
Top:    dcb6e21860214e7e40045b90d84a349de2618227
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-21 12:06:33 +0000

Refresh of 0001.patch

---

diff --git a/src/select.c b/src/select.c
index b85d9e3..94069b0 100644
--- a/src/select.c
+++ b/src/select.c
@@ -1025,7 +1025,7 @@ SelectObjectByName (int Type, char *Pattern, bool Flag)
         Cardinal i;
         LibraryEntryType *entry;
         ConnectionType conn;
-	
+
         if (menu->Name && REGEXEC (menu->Name))
           {
             for (i = menu->EntryN, entry = menu->Entry; i; i--, entry++)
